content/en/docs/concepts/overview/object-management-kubectl/kustomization.md

```diff
@@ -660,7 +668,7 @@ namePrefix: prod-
 EOF
 ```
 
-## How to apply/view/delete objects from Kustomization
+## How to apply/view/delete objects from Kustomize
```

Comment:
How to apply/view/delete objects using Kustomize

Reply:
nit, line 672, I guess you use kustomize the tool and kustomization.yaml interchangeably.
Use `--kustomize` or `-k` in `kubectl` commands to recognize ...

Reply:
updated